taxi/models.py

```diff
@@ -0,0 +1,40 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+    country = models.CharField(max_length=63)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=63, unique=True)
+
+    class Meta:
+        verbose_name = "driver"
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.username} ({self.first_name} {self.last_name})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=63)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="manufacturers",

```

Comment:
remind yourself how to name `related_name` properly
https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency

Reply:
done